optuna/cli.py

```diff
@@ -223,12 +224,45 @@ def take_action(self, parsed_args):
         return 0
 
 
+class StorageUpgrade(BaseCommand):
+    def get_parser(self, prog_name):
+        # type: (str) -> ArgumentParser
+
+        parser = super(StorageUpgrade, self).get_parser(prog_name)
+        return parser
+
+    def take_action(self, parsed_args):
+        # type: (Namespace) -> None
+
+        if self.app_args.storage is None and self.app_args.config is None:
+            raise CLIUsageError("Either --storage or --config option is required.")
+
+        config = optuna.config.load_optuna_config(self.app_args.config)
+        storage_url = get_storage_url(self.app_args.storage, config)
+
+        storage = RDBStorage(storage_url, skip_compatibility_check=True)
+        current_version = storage.get_current_version()
+        head_version = storage.get_head_version()
+        known_versions = storage.get_all_versions()
+        if current_version == head_version:
+            self.logger.info('This storage is up-to-date.')
+        elif current_version in known_versions:
+            self.logger.info('Upgrading the storage schema to the latest version.')
+            storage.upgrade()
+            self.logger.info("Completed to upgrade the storage.")
+        else:
+            self.logger.warn('Your optuna version seems outdated against the storage version. '
```

Comment:
Since `warn` is a deprecated method, please use `warning` instead.

Reply:
Thank you for pointing it out. 
I fixed this at the commit https://github.com/pfnet/optuna/pull/336/commits/43071e18f80f30a75d26d3badd4ef67c1538a050.